src/EventStore.Core/Telemetry/TelemetryService.cs

```diff
@@ -178,7 +178,11 @@ public sealed class TelemetryService : IDisposable,
 			message.Envelope.ReplyWith(new TelemetryMessage.Response("plugins", key, value));
 
 		foreach (var subsystem in _nodeOptions.PlugableComponents) {
-			subsystem.CollectTelemetry(collect);
+			try {
+				subsystem.CollectTelemetry(collect);
+			} catch (Exception ex) {
+				_log.Error(ex, "Failed to collect telemetry from a plugable component");
```

Comment:
Do we want to say which component failed? Also, is this an error or a warning?

Reply:
fair questions, i've adjusted the code